### PR TITLE
Update of portuguese translation

### PR DESCRIPTION
**Pull request by Kyoro93**
Improved translation context and corrected non-translated blocks

### CODE REVIEW

File: bundle_pt.properties [hunk 1]
```diff
@@ -115,20 +115,20 @@ supporter_status=Status de Apoiador
...
+a_gold_star_name_tag=Uma estrela de ouro ao lado do seu nome
+immunity_from_inactive_character_deletion=Imunidade contra exclusão de personagem por inatividade
+support_the_developer=Apoie o desenvolvedor <3
 
 # Settings
 
 video=Vídeo
 camera_zoom=Zoom da Câmera
-camera_zoom_description=The camera can be zoomed out to show more of the game world - this has a performance impact, especially on older devices. In addition, the lighting quality can be reduced to increase performance on old devices.
+camera_zoom_description=A câmera pode seer afastada para ver mais do mundo e isso tem impacto na performance, principalmente em dispositivos mais antigos. Além disso a qualidade da iluminação pode ser reduzida para melhorar a performance em disposistivos mais antigos
```
Comment (coelhucas) [inline bundle_pt.properties:128]:
```suggestion
camera_zoom_description=A câmera pode ser afastada para ver mais do mundo e isso tem impacto na performance, principalmente em dispositivos mais antigos. Além disso a qualidade da iluminação pode ser reduzida para melhorar a performance em dispositivos mais antigos
```

File: bundle_pt.properties [hunk 2]
```diff
@@ -217,29 +217,29 @@ are_you_sure_you_wish_to_leave_the_party_question=Tem certeza de que deseja sair
 invite=Convidar
 inviting_dots=Convidando...
 leave=Sair
-you_arent_invited_to_that_party=Você não foi convidade para esse grupo :(
+you_arent_invited_to_that_party=Você não foi convidado para esse grupo :(
```
Comment (coelhucas) [inline bundle_pt.properties:220]: There's no reason to have genre in this phrase

```suggestion
you_arent_invited_to_that_party=Você não recebeu um convite para este grupo :(
```

File: bundle_pt.properties [hunk 3]
```diff
@@ -217,29 +217,29 @@ are_you_sure_you_wish_to_leave_the_party_question=Tem certeza de que deseja sair
...
+they_are_already_invited=Eles já foram convidados :P
+too_many_open_invites=Você tem muitos convites em aberto :(
 
 # Character Details
 
 trade=Negociar
 whisper=Sussurrar
-invite_to_party=Convidar ao Grupo
-revoke_invite=Revogar Convite
+invite_to_party=Convidar pro Grupo
```
Comment (coelhucas) [inline bundle_pt.properties:234]: This is grammatically incorrect. The former is the better
```suggestion
invite_to_party=Convidar ao Grupo
```

File: bundle_pt.properties [hunk 4]
```diff
@@ -248,8 +248,8 @@ level=Nível
 
 # Moderate
 
-slap=Tapa
-kick=Chutar
+slap=Aviso
+kick=Kickar
```
Comment (coelhucas) [inline bundle_pt.properties:252]: The former makes more sense, we're localizing it not making up words
```suggestion
kick=Expulsar
```

**Review by coelhucas — COMMENTED** [review]
Let just a few nits after seeing this PR on Liam's stream. Is there a reason to replace spell (_feitiço_) with skill (_habilidade_)? The former looks more accurate IMO.

**Comment by carlislefox** [comment]
> 

Is this something you are able to rectify before I merge it in?

**Comment by coelhucas** [comment]
LGTM!

**Comment by Kyoro93** [comment]
Sorry for the delay
It's updated now =]

**Comment by carlislefox** [comment]
Hey buddy, fyi if the conflicts aren't resolved this month I'm going to close this ticket.

Cheers.

**Pull request closed without merging**